### ISSUE TITLE
hexagonal convolution

### ISSUE THREAD

**Issue by geoHeil** [uber/h3#242]
Hi,

https://www.youtube.com/watch?v=z3PaGIQTFSE greatly explains efficient spatial smoothing using H3 hexagons and convolutions.

Is `experimentalH3ToLocalIj` the convolution suitable coordinate system outlined in the video?

Can I find further information regarding this topic somewhere? This sounds interesting.

**Comment by chonguber** [comment]
Yes, `experimentalH3ToLocalIj` is the API used for the conversion. 

What other further information would you like to get on this topic?

**Comment by geoHeil** [comment]
Thanks @chonguber . 
In the video, you mention this runs on GPUs/pytorch. Do you also run this in a distributed computing environment spark)? When I do not require deep learning, for now, and just want the convolution operation for smoothing, how is an efficient way to implement this in spark, i.e. without causing lots of shuffles?
